.github/workflows/bump-version-release.yaml

```diff
@@ -95,3 +95,19 @@ jobs:
           github-token: ${{ secrets.github-token }}
           release-title: "${{ steps.bump-version.outputs.release-version }}"
           release-body: "${{ steps.build-changelog.outputs.single-changelog }}"
+
+      - name: Bump version
+        id: bump-version-snapshot
+        uses: bakdata/ci-templates/actions/patch-snapshot@feat/new-bumpversion
+        with:
+          working-directory: ${{ inputs.working-directory }}
+          test-version: ${{ steps.bump-version.outputs.release-version }}
+
+      - name: Commit and push changes including .bumpversion.cfg file
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.6.0
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          commit-message: "Bump version ${{ steps.bump-version.outputs.old-version  }} → ${{ steps.bump-version-snapshot.outputs.release-version }}"
```

Comment:
The example you linked produced wrong results:
- Bump version 0.0.4-snapshot → 0.0.4 
- Bump version 0.0.4-snapshot → 0.0.5-snapshot

but it should be:
- Bump version 0.0.4-snapshot → 0.0.4 
- Bump version 0.0.4 → 0.0.5-snapshot

Also, you added the second commit in this PR. Do we actually need this? Is there a requirement somewhere that specifies the desired result?

Reply:
I fixed the output. Regarding the second commit, I could not get it working properly without it

Reply:
I had 2 commits before